frontend/docs/admin-guide/install.md

```diff
@@ -969,6 +969,232 @@ policy. From the IAM Service page click "Policies" on the left hand side.
 13. After selecting the 3 policies click next and enter a name for the role. You can name the role whatever you'd like. Optionally add a description and tags and then click "Create role"
 14. Once the role has been created record the role ARN as we'll need to use it later.
 
+#### System Role
+
+1. The example below assumes you've created the permission boundary policy above in the [App Role](App Role) section and named the policy `mlspace-project-user-permission-boundary`. If you've named it something different
+you'll need to update the `"iam:PermissionsBoundary` condition in the `iam:CreateRole` statement below.
+2. After the permission boundary policy has been created you are now ready to create the application
+policy. From the IAM Service page click "Policies" on the left hand side.
+3. Create a new policy using the JSON editor and paste the following in the text area (after replacing the placeholder variables):
```

Comment:
It may be worth noting that they can use the app and notebook policies or if they want to restrict to least privileges they can use this one (saves them creating a new policy if they want to do that)

Reply:
Sure I'll make a note of that